### PR TITLE
Updates to developer_installation.rst

### PR DESCRIPTION
**Pull request by snesic**
Postgres version (12) still missing.

### CODE REVIEW

File: docs_manual/developer_installation.rst [hunk 1]
```diff
@@ -68,18 +68,25 @@ Clone the VarFish Server repository and switch into the checkout.
 Install Python Requirements
 ---------------------------
 
-With the conda/Python environment activated, install all the requirements.
+Some required packages have dependencies that are usually not preinstalled. Therefore, run
```
Comment (holtgrewe) [inline docs_manual/developer_installation.rst:71]: Each sentence goes to its own line.

**Review by holtgrewe — CHANGES_REQUESTED** [review]
> Postgres version (12) still missing.

Add this then please.

File: docs_manual/developer_installation.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ Install Postgres
 Follow the instructions for your operating system to install `Postgres <https://www.postgresql.org>`_.
 For Ubuntu, this would be::
 
-    sudo apt install postgresql
+    sudo apt install postgresql-12
```
Comment (holtgrewe) [inline docs_manual/developer_installation.rst:26]: Please rather add a note about this. Ubuntu 20 includes postgresql v12 as `postgresql` package.

**Review by holtgrewe — CHANGES_REQUESTED** [review]

File: docs_manual/developer_installation.rst (hunk 1 above)
Comment (snesic) [reply docs_manual/developer_installation.rst]: done

**Pull request merged**